scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -393,7 +393,7 @@ static struct task_ctx *lookup_task_ctx_may_fail(struct task_struct *p)
 	return bpf_task_storage_get(&task_ctxs, p, 0, 0);
 }
 
-static struct task_ctx *lookup_task_ctx(struct task_struct *p)
+static struct task_ctx *lookup_task_ctx_kay_fail(struct task_struct *p)
```

Comment:
Who's kay and why did they fail? 😆

Reply:
lol typo in the sed when I got tired of editing. 

funnily enough, the verifier was happy with and without the typo's.